aidatlu/test/tlu_test_configuration.yaml

```diff
@@ -1,59 +1,62 @@
 ################################################
-#
-# This configuration is only used during tests
-#
-###############################################
+#                                              #
+# This configuration is only used during tests #
+# Changing settings besides MOCK lead to fails #
+#                                              #
+################################################
 
+# Use mock I2C
+MOCK: True # This setting is ignored outside of tests
```

Comment:
Maybe one can use a pytest marker for that or environment variable. Let's discuss offline